framework/include/utils/MooseMeshUtils.h

```diff
@@ -289,4 +290,182 @@ void makeOrderedNodeList(std::vector<std::pair<dof_id_type, dof_id_type>> & node
                          std::vector<dof_id_type> & elem_id_list,
                          std::vector<dof_id_type> & ordered_node_list,
                          std::vector<dof_id_type> & ordered_elem_id_list);
+
```

Comment:
I think a new file is warranted.

Either MooseMeshElementConversionUtils (if you keep a references to a MooseMesh in a lot of the routines)
or just ElementConversionUtils if you remove the references to a MooseMesh